test/unit/test_task.rb

```diff
@@ -135,6 +138,18 @@ def expect_find_ruby_files
                 'features/featuresfake1.rb'])
   end
 
+  def expect_find_js_files
+    source_glob =
+      '{,{*,.*}.{js},' \
+      '{src,www,lib}/**/{*,.*}.{js}}'
+    expect_glob.with(source_glob)
+      .returns(['fake1.js',
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.